Cargo.toml

```diff
@@ -235,6 +235,7 @@ diplomat-tool = { git = "https://github.com/rust-diplomat/diplomat", rev = "8744
 # EXTRA_CAPI_DEPS
 # EXTRA_BLOB_DEPS
 # EXTRA_FS_DEPS
+arrayvec = { version = "0.7.2", default-features = false }
```

Comment:
issue: we already have `smallvec` in this list, can you use that?

Reply:
I think `arrayvec` is slightly better than `smallvec` in this instance because there is a known, small upper limit to the length of the vector. `smallvec` has code and branching to overflow to the heap, which we don't need here.

`arrayvec` is in Cargo.lock so it's not a new dependency.

Reply:
It is not currently a runtime dep: https://github.com/unicode-org/icu4x/blob/main/tools/make/depcheck/src/allowlist.rs#L41

Reply:
True. Flagging @Manishearth for the new runtime dependency.

Reply:
If we're adding another crate on top of `smallvec`, I think it should be `heapless`, which has more features than `arrayvec`. I'm not convinced this is worth adding another crate though.